tests/unit/scp_test.py

```diff
@@ -57,3 +59,44 @@ def test_get(dst_path, src_path, ssh_scp, transmit_payload):
     """Check that SCP file download works."""
     ssh_scp.get(str(src_path), str(dst_path))
     assert dst_path.read_bytes() == transmit_payload
+
+
+@pytest.fixture
+def dst_path_exists(tmp_path):
```

Comment:
This name sounds like a check, but it's not. It should be renamed to reflect what it actually does.

Reply:
Moreover, it mentions that something exists, but the code uses `non-existing` in the name. Which one is it? I'm confused.

Reply:
```suggestion
def pre_existing_file_path(tmp_path):
```